common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -132,13 +132,20 @@ public class RssClientConf {
           .defaultValue(0)
           .withDescription("Number of threads used in the client thread pool.");
 
-  public static final ConfigOption<Boolean> NETWORK_CLIENT_PREFER_DIRECT_BUFS =
+  public static final ConfigOption<Boolean> NETTY_CLIENT_PREFER_DIRECT_BUFS =
```

Comment:
NETWORK_CLIENT_PREFER_DIRECT_BUFS
->
NETTY_CLIENT_PREFER_DIRECT_BUFS

This is more suitable.